common-pg/src/main/scala/com/socrata/pg/store/index/FullTextSearch.scala

```diff
@@ -23,19 +24,29 @@ trait FullTextSearch[CT] {
     val phyCols = repsSearchableTypes.flatMap(rep => rep.physColumns.map(phyCol =>
       coalesce(ctx.map(qualify(phyCol, _)).getOrElse(phyCol))
     ))
-    toTsVector(phyCols)
+    toTsVector(phyCols.zip(phyCols))
   }
 
-  def searchVector(selection: OrderedMap[ColumnName, CoreExpr[_, CT]], ctx: Option[Context]): Option[String] = {
-    val cols = selection.view.filter(x => SearchableTypes.contains(x._2.typ)).flatMap { case (columnName, expr) =>
+  def searchVector(selection: OrderedMap[ColumnName, CoreExpr[UserColumnId, CT]], ctx: Option[Context]): Option[String] = {
```

Comment:
Was this type previously anonymous for a specific reason?

Reply:
🤔 I don't know, but I thought the compiler would tell me if there were unsafe uses of it. I could be cautious and leave it anonymous so if there are any that don't match, it just goes to the case statement default.

Reply:
It just didn't care about that type before

Reply:
With type erasure I don't think the compiler would be able to complain in this case. Don't know for sure though.

Reply:
removing the type here gives a type erasure error on the case match line. I'm learning more about it, but @rjmac you're saying its ok to leave the type match in the function head here?

Reply:
Scala doesn't do matching in function heads.  So, yes.  Anything that called it with the wrong type would complain.